### PR TITLE
Regenerate give site on move

### PR DESCRIPTION
**Pull request by twinge**
@jbirdjavi

### CODE REVIEW

File: app/models/sp_application.rb [hunk 1]
```diff
@@ -12,7 +12,6 @@ class SpApplication < Fe::Application
   COST = 25
 
   before_create :create_answer_sheet_question_sheet
-  after_save :complete
```
Comment (jbirdjavi) [inline app/models/sp_application.rb]: Is this intentional?

Comment (twinge) [inline app/models/sp_application.rb]: Yes. It was duplicated down below

Comment (jbirdjavi) [inline app/models/sp_application.rb]: Ahh okay.  Had to break out the whole file view to see it.  :+1:

Comment (twinge) [inline app/models/sp_application.rb]: Yes. It was duplicated down below

**Pull request merged**